src/main/scala/org/virtuslab/akkasaferserializer/StandardCodecs.scala

```diff
@@ -1,17 +1,34 @@
 package org.virtuslab.akkasaferserializer
 
+import akka.actor
+import akka.actor.typed.{ActorRef, ActorRefResolver, ActorSystem}
+import akka.serialization.Serialization
+import akka.stream.{SinkRef, SourceRef, StreamRefResolver}
 import io.bullet.borer.Codec
 
 import java.io.{ByteArrayInputStream, ByteArrayOutputStream, ObjectInputStream, ObjectOutputStream}
 import java.time.OffsetDateTime
 
 object StandardCodecs {
+
+  implicit def sinkRefCodec[T](implicit system: actor.ActorSystem = serializationSystem): Codec[SinkRef[T]] = {
```

Comment:
Gr8, lemme quickly check on my commercial project (let's codename it Hydra from now on...)

Reply:
Whoops I'm blocked on this problem with type aliases (since borer 1.7.1 is not yet available for Scala 2.12, only 2.13)... let's leave as is for now